src/main.cpp

```diff
@@ -47,44 +49,48 @@ int main(int argc, char** argv)
 	SDL_Event e;
 
 	while (playing) {
-		currentTime = SDL_GetTicks();
-		elapsedTime = currentTime - oldTime;
-		oldTime = currentTime;
-
+		// EVENT HANDLING
 		while (SDL_PollEvent(&e) != 0) {
 			if (e.type == SDL_QUIT) {
 				playing = false;
 			} else if (e.type == SDL_MOUSEBUTTONDOWN) {
 			}
 		}
 
+
 		// GAME LOGIC
 
-		debugCounter += elapsedTime;
-		if (DEBUG_OUTPUT && debugCounter > 1000) {
-			debugCounter -= 1000;
-			debugTick = true;
-		}
 
 		// RENDERING
 		frogRenderer.clearScreen();
+		frogRenderer.renderThing(splashThing, 0, 0);// Render Splash Logo
+		frogRenderer.presentScreen();
 
-		// Render Splash Logo
-		frogRenderer.renderThing(splashThing, 0, 0);
 
-		frogRenderer.presentScreen();
+		#if DEBUG_OUTPUT // DEBUG INFO
```

Comment:
Good call on moving the debug logic together. Just for readability, let's move the `if` macro down a line or two for readability.